app/components/Inputs/StyledReactSelect/StyledReactSelect.jsx

```diff
@@ -69,6 +70,8 @@ const customStyles = {
     return {
       ...styles,
       fontSize: props.selectProps.fontSize,
+      fontWeight: props.selectProps.fontWeight,
+      [props.selectProps.textAlign]: 0,
```

Comment:
Aligns the single value to the right.
Before:
<img width="158" alt="screen shot 2018-11-10 at 10 33 42" src="https://user-images.githubusercontent.com/486954/48299684-0ce26580-e4d9-11e8-8db8-8d7855b9617d.png">

After:
<img width="158" alt="screen shot 2018-11-10 at 10 33 16" src="https://user-images.githubusercontent.com/486954/48299685-110e8300-e4d9-11e8-9133-60ca9913b4b6.png">